component-matrix/pom.xml

```diff
@@ -88,7 +88,7 @@
         <version.org.jboss.logging.jboss-logging>3.3.1.Final</version.org.jboss.logging.jboss-logging>
         <version.org.jboss.logging.jboss-logging-tools>2.1.0.Final</version.org.jboss.logging.jboss-logging-tools>
         <version.org.jboss.logging.jul-to-slf4j-stub>1.0.1.Final</version.org.jboss.logging.jul-to-slf4j-stub>
-        <version.org.jboss.logmanager.jboss-logmanager>2.1.0.Final</version.org.jboss.logmanager.jboss-logmanager>
+        <version.org.jboss.logmanager.jboss-logmanager>2.1.1.Final</version.org.jboss.logmanager.jboss-logmanager>
```

Comment:
is there a WFCORE issue for this component upgrade?

Reply:
No. I guess I should probably create one and make this a separate commit. I put it in with this commit because it's got a required change to make the resource work.